erdjs/src/smartcontracts/typesystem/types.ts

```diff
@@ -34,9 +34,12 @@ export class Type {
         return this.typeParameters[0];
     }
 
+
+    /**
+     * Generates type expressions similar to elrond-wasm-rs. 
+     * Involves recursive calls to toString().
```

Comment:
Might not be necessary to specify `Involves recursive calls to toString().`

Reply:
Removed.